src/stream/src/executor/backfill/cdc/cdc_backfill.rs

```diff
@@ -101,6 +96,11 @@ impl<S: StateStore> CdcBackfillExecutor<S> {
             pk_in_output_indices.len() + METADATA_STATE_LEN,
         );
 
+        let options = scan_options.unwrap_or(CdcScanOptions {
+            disable_backfill,
+            ..Default::default()
+        });
```

Comment:
I would suggest moving this into `from_proto` so that we can accept a single field of `scan_options: CdcScanOptions` in `new`.